src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/DecisionRates.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.sscs.domain.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+@JsonInclude(value = JsonInclude.Include.NON_EMPTY)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DecisionRates {
```

Comment:
Think this can be removed

Reply:
OnlineHearing data object is used by both MYA and COR, can't removed it unless if we modify the frontend